k8s/kafka/operator/deployment-script.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
-wget https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.35.1/strimzi-0.35.1.tar.gz
-tar -zxvf strimzi-0.35.1.tar.gz
```

Comment:
We shall need to watch this closely as sometimes these version things can be chaotic! @BenjaminSsempala